src/views/Ifos/hooks/v1/useGetPublicIfoData.ts

```diff
@@ -74,9 +73,43 @@ const useGetPublicIfoData = (ifo: Ifo): PublicIfoData => {
     }))
   }, [address, currentBlock, releaseBlockNumber])
 
+  const fetchInActiveIfoData = useCallback(async () => {
```

Comment:
@memoyil not entirely sure what's the point of this. It seems to make same amount of calls as `fetchIfoData`, the only difference is some blocks/progress calculations are omitted. I think it might be better to use `fetchIfoData` for readability sake. Unless I miss something 🤔

Reply:
also nitpick -  `Inactive` instead of `InActive` 😄 
if we even need it 🤔

Reply:
I understand your point about `useBlocks` updating v2 IFOs.
I'm just thinking the fetchIfoData can be used for finished IFOs too. If you wanna improve performance with removing progress calculations maybe we could just put a check for that inside `fetchIfoData`. The `isActive` can be passed as an argument to it to make it easier on `useCallback` dep array

Reply:
> 

Done